apps/main/src/dao/components/PageAnalytics/CrvStats/index.tsx

```diff
@@ -39,7 +41,7 @@ const CrvStats: React.FC = () => {
             title={t`Total CRV`}
             data={
               <MetricsColumnData>
-                {noProvider ? '-' : formatNumber(veCrvData.totalCrv, { notation: 'compact' })}
+                {noProvider ? '-' : formatNumber(Number(veCrvData.totalCrv) / 10 ** 18, { notation: 'compact' })}
```

Comment:
Why do we have `10**18` all over the code instead of the parsing code?

Reply:
See Slack discussion wrt formatting values from the store. As suggested, I could create a util function (`fromGwei`), but I'd prefer to implement that in another PR, together with some other global fixes that I need to apply as a result of the other refactor PRs and their review feedback.